shell/components/ExplorerProjectsNamespaces.vue

```diff
@@ -235,6 +237,21 @@ export default {
     }
   },
   methods: {
+    /**
+     * Check if resource contains PSA labels
+     * @param {*} row HciNamespace
+     */
+    hasPSA(row) {
+      return hasPSALabels(row);

```

Comment:
You may want to use the function outside of the model, so you don't have to import everything.
For instance, you are in this case into the namespace, not in the PSA. How do you deal with that?

Reply:
`shell/models/namespace.js` 

```
  get hasPSALabels() {
    return hasPSALabels(row); 
  }
```

Though i think we can remove the need to use it outside of the models

Reply:
As we have a parameter, I'm force to turn into a simple method. 

<img width="395" alt="Screenshot 2022-12-12 at 15 45 04" src="https://user-images.githubusercontent.com/5009481/207074619-7be26cf0-10e2-4b62-b1cc-3496d09025eb.png">

Reply:
As we have a parameter, I'm force to turn into a simple method. 

<img width="395" alt="Screenshot 2022-12-12 at 15 45 04" src="https://user-images.githubusercontent.com/5009481/207074619-7be26cf0-10e2-4b62-b1cc-3496d09025eb.png">

Reply:
Sorry, typo above. The `row` is the instance available on the model. so it would be...

```
  get hasPSALabels() {
    return hasPSALabels(this); 
  }
```